RandLAPACK/gpu_functions/rl_cusolver.hh

```diff
@@ -0,0 +1,61 @@
+#ifndef bpcg_cublas_h
+#define bpcg_cublas_h
```

Comment:
Use ``#pragma once``.

Please remove the explicit include guard. If you have to keep it, please give the macro a better name.

Reply:
Done.